.golangci.yaml

```diff
@@ -14,29 +14,12 @@ linters:
     - prealloc
     - unconvert
     - unparam
-    - govet
-    - deadcode
-    - gosimple
-    - ineffassign
-    - staticcheck
-    - structcheck
-    - typecheck
-    - unconvert
-    - unused
-    - varcheck
+  disable:
```

Comment:
I think that's for the GH-Action for `golangci-lint` is using it:
https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/.github/workflows/knative-style.yaml

Reply:
Yes, GH-action is using it.

There were some rules that simply failed because of the generated files. I don't know how this code came all this way with these rules.

I disabled the rules that don't exist in knative/eventing.